sections/multicolumn.liquid

```diff
@@ -88,8 +96,8 @@
         {%- endfor -%}
```

Comment:
I would say that because this is existing on main and we may change that, we can do it in a follow up depending on what route we take 🤔 Issue here: https://github.com/Shopify/dawn/issues/1397